web/src/components/molecules/View/viewMenuItem.tsx

```diff
@@ -71,31 +71,19 @@ const ViewsMenuItem: React.FC<Props> = ({
   ];
 
   return (
-    <StyledDropdownButton
-      trigger={["click"]}
-      type="text"
-      icon={<Icon icon="more" />}
-      menu={{ items: children }}>
+    <Wrapper>
       {t(view.name)}
```

Comment:
Not related to your PR but can you please fix it? we don't need t() in here.
```suggestion
      {view.name}
```

Reply:
Yes, sure!